app/src/main/AndroidManifest.xml

```diff
@@ -11,6 +11,8 @@
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" /> <!-- Used by sentry. -->
     <uses-permission android:name="com.android.launcher.permission.INSTALL_SHORTCUT" />
     <uses-permission android:name="android.permission.FOREGROUND_SERVICE"/>
+    <!-- Needed to prompt the user to give permission for camera access -->
```

Comment:
nit: Needed to prompt the user to give permission when uploading images from the camera.

Reply:
I wonder if we can make this an optional feature in the component? Giving camera permission to Focus may look odd? Maybe file a follow-up? 🤔

Reply:
Yeah, that's a good point. I filed https://github.com/mozilla-mobile/android-components/issues/7399 to investigate using a runtime permission instead.